content/en/contribute/_index.md

```diff
@@ -7,111 +7,127 @@ menu:
     weight: 10
 ---
 
-The Cloud Native Glossary content is stored in [this GitHub repo](https://github.com/cncf/glossary) 
-where you'll find a list of [issues](https://github.com/cncf/glossary/issues), [PRs](https://github.com/cncf/glossary/pulls), and 
-[discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
+## Welcome 
 
-There are four ways you can contribute:
+Welcome to the Cloud Native Glossary contributing guide and thank you for your interest. If you would like to contribute to our project, checkout the following list of contribution types and their corresponding sections in this guide: 
 
 1) [Work on an existing issue](#work-on-an-existing-issue)
 2) [Propose new terms](#propose-new-terms)
 3) [Update existing ones](#update-an-existing-term)
-4) [Help translate the glossary](#help-translate-the-glossary)
+4) [Localize the glossary](#help-translate-the-glossary)
+
+## CNCF glossary overview 
+
+The goal of this glossary is to simplify the cloud native space — which is notorious for its complexity — and thus make it more accessible to people. 
+
+The Cloud Native Glossary content is stored in [this GitHub repo](https://github.com/cncf/glossary) 
+where you'll find a list of [issues](https://github.com/cncf/glossary/issues), pull requests ([PRs](https://github.com/cncf/glossary/pulls)), and 
+[discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
 
 ## Who can contribute?
 
-The goal of this Glossary is to make the cloud native space — which is notorious for its complexity — simpler for people by making it easier to understand. 
-To do that, we must focus on simplicity. 
+Basically, anyone can contribute, however, how you participate in the project depends on your level of expertise of the cloud native space. 
+Simplifying complex concepts requires a deep knowledge of the topic. Therefore, to contribute with a new term you must be proficient in the cloud native space. 
+
+The localization effort is where beginners can make valuable contributions to the glossary. 
+With solid existing definitions in English, less experienced contributors can localize terms to a target language. You can join an existing localization team or create a new one. Read the section [Help translate the glossary](#help-translate-the-glossary) of this guide to learn how to get started. 
+
+## Before you start
```

Comment:
Not sure how to do this, but external links should open in a new tab. @seokho-son @iamNoah1, can you help?

Reply:
'shift + click on the link' ? :)

Reply:
FYI we no longer force people to open external links in new tabs. I wrote up some reasons for that which I’ll copy below:
As part of the accessibility improvements we’ve been working on for our websites, we’d like to get away from forcing external sites to open in new browser tabs. The idea behind this was originally to prevent people from leaving our site or at least giving them an easy way to get back to our site by leaving the tab open. This practice, however, is not recommended for a few reasons:
- It can be disorienting to screen reader users, those with cognitive impairments or people who magnify their screen
- In a lot of cases it actually makes it harder to go back to the page you left since the Back button won’t take you there. This affects all users and is exacerbated on mobile where the old window is never visible.
- Opening a new tab takes control out of the user’s hands
[See this article for a full analysis](https://www.nngroup.com/articles/new-browser-windows-and-tabs/).

Reply:
Just FYI:
we cannot achieve this (make the link open in new tab) with markdown, so we have to use HTML if we want to.

Ref: https://stackoverflow.com/questions/4425198/can-i-create-links-with-target-blank-in-markdown